src/main/java/ch/njol/skript/expressions/ExprItemAmount.java

```diff
@@ -19,81 +19,82 @@
 package ch.njol.skript.expressions;
 
 import ch.njol.skript.aliases.ItemType;
-import ch.njol.skript.classes.Changer;
 import ch.njol.skript.classes.Changer.ChangeMode;
 import ch.njol.skript.doc.Description;
 import ch.njol.skript.doc.Examples;
 import ch.njol.skript.doc.Name;
 import ch.njol.skript.doc.Since;
 import ch.njol.skript.expressions.base.SimplePropertyExpression;
 import ch.njol.skript.util.slot.Slot;
-import ch.njol.util.coll.CollectionUtils;
 import org.bukkit.event.Event;
+import org.bukkit.inventory.ItemStack;
 import org.eclipse.jdt.annotation.Nullable;
 
 @Name("Item Amount")
 @Description("The amount of an <a href='classes.html#itemstack'>item stack</a>.")
 @Examples("send \"You have got %item amount of player's tool% %player's tool% in your hand!\" to player")
-@Since("2.2-dev24")
+@Since("2.2-dev24, INSERT VERSION (itemstacks)")
```

Comment:
I don't think this should have a version comment since it's fixing (presumably) intended behaviour that is missing rather than adding new, previously unintended behaviour.